homeassistant/components/samsungtv/remote.py

```diff
@@ -0,0 +1,41 @@
+"""Support for the SamsungTV remote."""
+from __future__ import annotations
+
+from collections.abc import Iterable
+from typing import Any
+
+from homeassistant.components.remote import ATTR_NUM_REPEATS, RemoteEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import SamsungTVEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the Samsung TV from a config entry."""
+    bridge = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities([SamsungTVRemote(bridge=bridge, config_entry=entry)], True)
+
+
+class SamsungTVRemote(SamsungTVEntity, RemoteEntity):
+    """Device that sends commands to a SamsungTV."""
+
+    async def async_turn_off(self, **kwargs: Any) -> None:
+        """Turn the device off."""
+        await self._bridge.async_power_off()
+
```

Comment:
Should this entity be marked as `assumed_state`, since it doesn't know its state?

Reply:
It could, but then we also need to update to the assumed state. Currently the state is always unknown.